tests/drivers/spi/spi_loopback/testcase.yaml

```diff
@@ -26,15 +26,15 @@ tests:
       - CONFIG_SPI_ASYNC=n
     filter: CONFIG_SOC_FAMILY_STM32
     platform_allow: nucleo_g474re nucleo_f207zg nucleo_f429zi nucleo_f746zg nucleo_wb55rg
-        nucleo_l152re nucleo_wl55jc nucleo_h743zi
+        nucleo_l152re nucleo_wl55jc nucleo_h743zi b_l4s5i_iot01a
```

Comment:
Can we use disco_l475_iot1 instead ?
(since available on the bench, which is not the case of b_l4s5i_iot01a)

Reply:
ok, changed to disco disco_l475_iot1 ' (no DMAMUX)